pkg/controller/machineset/controller_test.go

```diff
@@ -262,3 +266,67 @@ func TestAdoptOrphan(t *testing.T) {
 		}
 	}
 }
+
+func TestMachineSetReconcile(t *testing.T) {
```

Comment:
Any reason not to use Ginkgo for these two tests as well?

```
var _ = Describe("MachineSet Reconcile", func() {
  var r reconciler
  var result reconile.Result
  var reconcileErr error

  BeforeEach(func() {
    // set up reconciler as r with scheme and new fake recorder
  })

  JustBeforeEach(func() {
    result, reconcileErr = r.Reconcile(request)
  })

  Context("with a valid machineset", func() {
    BeforeEach(func() {
       // set up machinset
       // set up fake client
       // override r.client
    })

    It("has the expected result", func() {
      Expect(result).To(Equal())
     })

    It("has the expected error", func() {
      Expect(reconcileErr).To(Equal())
     })

    It("did something with events", func() {
      g.Eventually(r.recorder.Events).Should(Receive())
     })
  })

  Context("with an invalid machineset", func() {
    // This should look basically the same as above
  })
})

```

Reply:
👍